authentication/tests.py

```diff
@@ -147,7 +150,8 @@ class SendRecoverPasswordEmailTest(TestCase):
     def setUp(self):
```

Comment:
the setUp function seems to be repeating, can you create a superclass so that each case have the same setup and not needing to be written more than once.

Reply:
Ok, got it